src/backend/database_models/agent_task.py

```diff
@@ -0,0 +1,42 @@
+from sqlalchemy import (
+    DateTime,
+    ForeignKey,
+    Integer,
+    LargeBinary,
+    String,
+    Text,
+    UniqueConstraint,
+)
+from sqlalchemy.orm import Mapped, mapped_column
+
+from backend.database_models.base import Base, MinimalBase
+
+
+class SyncCeleryTaskMeta(MinimalBase):
+    __tablename__ = "sync_celery_taskmeta"
+
+    id: Mapped[int] = mapped_column(Integer, primary_key=True)
+    task_id: Mapped[str] = mapped_column(String(155), unique=True)
+    status: Mapped[str] = mapped_column(String(50))
+    result: Mapped[bytes] = mapped_column(LargeBinary)
+    date_done: Mapped[DateTime] = mapped_column(DateTime)
+    traceback: Mapped[str] = mapped_column(Text)
+    name: Mapped[str] = mapped_column(String(155))
+    args: Mapped[bytes] = mapped_column(LargeBinary)
+    kwargs: Mapped[bytes] = mapped_column(LargeBinary)
+    worker: Mapped[str] = mapped_column(String(155))
+    retries: Mapped[int] = mapped_column(Integer)
+    queue: Mapped[str] = mapped_column(String(155))
+
+
+class AgentTask(Base):
+    __tablename__ = "agent_tasks"
+
+    agent_id: Mapped[str] = mapped_column(
+        ForeignKey("agents.id", ondelete="CASCADE"), nullable=False
+    )
+    task_id: Mapped[str] = mapped_column(Text, nullable=False)
```

Comment:
Should we make this a FK?

Reply:
ok few findings.

its seems to be a bit tricking referring to files in the celery table during a migration. it seems like this is largely due to the fact that the celery table is not created as part of our migration system. 

i tried looking into ways to work around but i dont think its worth it for now. this is working as is and we can revisit this later (or we potentially move all this to redis regardless).